tally.py

```diff
@@ -0,0 +1,24 @@
+'''
+ @file      tally.py
+ @brief     Contains the Tally class
+ @author    Luke Eure
+ @date      October 13, 2015
+'''
+
+import numpy as np
+
+'''
+ @class Tally tally.py "tally.py"
+ @brief Tally contains tallies of information used in neutron simulation
+'''
+
+class Tally():
+    def __init__(self):
+        self.count = 0
+    def increment(self, amt):
```

Comment:
I'm getting picky here, but normally when there's an increment function it means "add one". I would prefer the function to be named ``add``